ui.apps/src/main/content/jcr_root/apps/saas-aem-module/clientlibs/wknd-sample/.content.xml

```diff
@@ -2,4 +2,7 @@
 <jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
```

Comment:
@nhirrle @kevinolivar isn't this generated by ui.frontend webpack?
Considering the maven modules order (ui.frontend being before ui.apps), imo we should completely remove the clinetlib, since it is generated on a target path under ui.apps.

Reply:
Hi @igorsimovski ,

agree, wknd.sample is indeed generated and copied over from ui.frontend. Anything auto-generated shouldn't be part of the codebase IMHO.

Did I commit this folder at the first place? Or it was to replaced the existing original one (I did not have control over its generation).

Reply:
Hey @kevinolivar, it's been initially committed by me :), but at that time it was not generated by webpack.
I think that with the introduction of the its implementation in ui.frontend we should've removed the initial clinetlib definition from ui.apps

- [ ] Please remove ui.apps/src/main/content/jcr_root/apps/saas-aem-module/clientlibs/wknd-sample

Reply:
It's all good I deleted the wknd lib